packages/react-table/src/components/Table/SortColumn.tsx

```diff
@@ -42,6 +49,34 @@ export const SortColumn: React.FunctionComponent<SortColumnProps> = ({
   } else {
     SortedByIcon = ArrowsAltVIcon;
   }
+
+  if (onFavorite) {
+    return (
+      <ActionList isIconList>
+        <ActionListItem>
+          <Button
+            variant="plain"
+            icon={<StarIcon />}
+            aria-label={favorited ? 'Unfavorite all' : 'Favorite all'}
```

Comment:
This works if we simply want a bi-state choice for this button. One issue at least in the example is that if I individually click each row's favorite button, I would expect this "favorite all" button to update accordingly to say "Unfavorite all". Right now it still says "Favorite all" even if all of the rows are favorited individually.

Another thing is that we really shouldn't hardcode these labels. This is what we're doing with the MenuItem's favoritable logic, but that should be updated as well. Not a total blocker here since this is how we're doing it elsewhere.

Reply:
By "hardcoding the labels" you mean they should be customizable? So that the string is not harcoded in the `SortColumn.tsx` directly, but instead the string should be part of the `TableFavoritable` example and should be passed via props?

Reply:
Correct. Ideally we should allow consumers to customize the labels for whichever reason (verbiage preference or i18n)